diagnostics/task_manager_dump.md

```diff
@@ -0,0 +1,15 @@
+# Crash Dump from Task Manager
+Normally, crash dumps are gathered automatically and available in the app's user data folder (see [Crash Dumps](crash.md)). In some cases, a user might need to manually gather a crash dump, normally for unresponsive processes. To do so:
+
+1. Open Task Manager (Ctrl+Shift+Esc).
+2. Click "Details" on the left menu.
+3. Sort by the "Name" column.
+4. Scroll to find "msedgewebview2.exe"
+5. To find the right msedgewebview2.exe, you can use a combination of:
+    1. The "Package name" column, which will represent the application using WebView2.

```

Comment:
my ask manager does not have "package name" column to show

Reply:
Columns can be enabled/disabled. We might need to add instructions or link.
![image](https://github.com/user-attachments/assets/1ba046b9-0aef-46fc-99d8-d436cc33df36)

Reply:
found by following steps